sdk/storage/Azure.Storage.Blobs/api/Azure.Storage.Blobs.netstandard2.0.cs

```diff
@@ -388,6 +388,10 @@ public partial class BlobCorsRule
         public string ExposedHeaders { get { throw null; } set { } }
         public int MaxAgeInSeconds { get { throw null; } set { } }
     }
+    public enum BlobDeleteType
```

Comment:
We need to hide this in the swagger readme.
```
### Hide BlobDeleteType
``` yaml
directive:
- from: swagger-document
  where: $.parameters.BlobDeleteType
  transform: >
    $["x-az-public"] = false;
```
```